corehq/apps/userreports/reports/view.py

```diff
@@ -356,6 +356,10 @@ def excel_response(self):
         return file
 
 
+class CustomConfigurableReport(ConfigurableReport):
```

Comment:
I think since this is non-intuitive it might be worth having either a comment or a doc string explaining why this class exists